trackstop.lua

```diff
@@ -236,7 +260,170 @@ function RollerOverlay:init()
   }
 end
 
+selected_stop = selected_stop or nil
```

Comment:
why does this need to be a global instead of an instance variable in ReorderStopsWindow?

Reply:
Can fix!

Reply:
I guess with the way modals work I don't need the other global variable either. Nice. Removed that too.